packages/conf-clang-format/conf-clang-format.1/opam

```diff
@@ -0,0 +1,19 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+homepage: "https://clang.llvm.org/docs/ClangFormat.html"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+authors: "The Clang Team"
+license: "NCSA"
+build: ["clang-format" "--version"]
+depexts: [
+  ["clang-format"] {os-family = "debian"}
+  ["clang-tools-extra"] {os-family = "fedora"}
+  ["clang15-extra-tools"] {os-distribution = "alpine"}
```

Comment:
I am not convinced by this, in the sense that previous alpine would work fine if we were selecting the right versions: `alpine-3.16` provides `clang-extra-tools` which installs version 13.0.1.
As per macports, I think it may be better to omit this, but I'll wait to see what the other maintainers think before doing anything

Reply:
I agree the `alpine` match was not ideal, but I was not sure how to proceed. Should I specify a specific version of alpine, such as this?

```
  ["clang-extra-tools"] {os-distribution = "alpine" & os-version < "3.17" }
  ["clang15-extra-tools"] {os-distribution = "alpine" & os-version >= "3.17" }
```

There is something similar in the conf-postgresql.1 package (postgresql-dev/postgresql14-dev).

I have a Mac with homebrew for testing, so I'm afraid I cannot easily test MacPorts without risking breaking things. But if there is a way, I can try it.

Reply:
```suggestion
  ["clang-extra-tools"] {os-distribution = "alpine" & os-version < "3.17" }
  ["clang15-extra-tools"] {os-distribution = "alpine" & os-version >= "3.17" }
```
Let's try your suggestion here